pkg/cqrs/sqlitecqrs/migrations/000001_create_apps_table.up.sql

```diff
@@ -27,6 +27,11 @@ CREATE TABLE functions (
 -- XXX: This does not conform to the cloud.  It only includes basic fields.
 CREATE TABLE events (
 	internal_id BLOB, -- cannot use CHAR(26) for ulids, nor primary keys for null ter
+	account_id CHAR(36),
+	workspace_id CHAR(36),
+	source VARCHAR(255),
+	source_id CHAR(35),
+	received_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
why is this migration modified instead of having a separate migration?
is sqlite only expect to work in memory and never on the filesystem?

Reply:
Right now we have no persistence so this is in general safe